src/components/style/footer.css

```diff
@@ -0,0 +1,102 @@
+/* all fixed values are from marketing site footer */
+.FDSFootercontainer {
```

Comment:
the naming convention we use for classes is like this `.fdsButton`, `.fdsButton--m`, `.fdsButton-label`.
do you think you could update them to match this?